aip/general/0126.md

```diff
@@ -55,12 +55,18 @@ message Book {
   - An exception to this rule is if there is a clearly useful zero value. In
     particular, if an enum needs to present an `UNKNOWN`, it is usually clearer
     and more useful for it to be a zero value rather than having both.
+- The other values **should not** be prefixed by the name of the enum itself.
+  This generally requires users to write `MyState.MYSTATE_ACTIVE` in their
+  code, which is unnecessarily verbose.
+  - Note that some languages (including C++) hoist enum values into the parent
+    namespace, which can result in conflicts for enums with the same values. To
+    avoid this, multiple enums in the same namespace **must not** share any
+    values. To avoid sharing values, APIs **may** prefix enum values with the
+    name of the enum. In this case, they should do so consistently within the
```

Comment:
The "should" here should be in bold, and I'd be tempted to make it a must.

Reply:
Done.

Reply:
> I can't see a way around that, unfortunately.

I agree, and overall, I wish there were a way to make this guidance a little simpler:( But I think this is the price we pay to get the (admittedly nice) benefit of terse enum names.